### ISSUE TITLE
Cannot read property 'cssClass' of undefined

### ISSUE THREAD

**Issue by justinbwarner** [krzysu/flot.tooltip#136]
Getting an error with version 0.8.6 that didn't happen in 0.8.5. Created a plunker that reproduces the issue:

https://embed.plnkr.co/snAKVOxXDSEobEdr81WK/

Cannot read property 'cssClass' of undefined
at FlotTooltip.getDomElement (jquery.flot.tooltip.js:329)
at Plot.FlotTooltip.init.plot.removeTooltip (jquery.flot.tooltip.js:320)
at jquery.flot.tooltip.js:116
at executeHooks (jquery.flot.js:717)
at Plot.shutdown (jquery.flot.js:1363)
at setupCanvases (jquery.flot.js:1327)
at new Plot (jquery.flot.js:707)

**Comment by krzysu** [comment]
I see there was an API change, and backwards compatibility is not written perfectly ;)

just try using new configuration object:

instead of
```
tooltip: true,
tooltipOpts: {
    content: "%s %x, %y"
}
```

use
```
tooltip: {
    show: true,
    content: "%s %x, %y"
}
```

let me know if it helps!

**Comment by loren138** [comment]
This error seems to occur if you have flot.tooltip.js installed, but do not have tooltips turned on for a particular graph. (ie show: false)

**Comment by krzysu** [comment]
I agree with breaking changes and versioning, unfortunately I'm not an active maintainer of this repo, talk with @Roundaround

**Comment by po5i** [comment]
Seems that can be fixed with #138

**Comment by krzysu** [comment]
#138 merged and released as 0.8.7, let me know if everything is fine
